src/mapping/contentful/mappings.ts

```diff
@@ -1,4 +1,5 @@
 import dynamic from 'next/dynamic';
+import { CtfTeaserLinkedGql } from '@src/data/contentful/teaser-linked/ctf-teaser-linked-gql';
```

Comment:
Are you sure this is correct, seems like all the other imports for gql are not requried